proto/nexus/v1beta1/query.proto

```diff
@@ -54,7 +54,7 @@ message FeeResponse { exported.v1beta1.FeeInfo fee_info = 1; }
 message TransferFeeRequest {
   string source_chain = 1;
   string destination_chain = 2;
-  cosmos.base.v1beta1.Coin amount = 3 [ (gogoproto.nullable) = false ];
+  string amount = 3;
```

Comment:
Why change it to string?

Reply:
The auto-generated REST endpoint doesn't support parsing custom types. And it's also easier for the frontend to use a string input. Cosmos SDK queries follow a similar model. After discussion with Chris, we discussed on adding a method on the request type to return the parsed amount instead.

Reply:
So is it fair to say that in the request objects of queries/txs, only protobuf primitives should used?

Reply:
It's unclear if submitting it in the GET request body could work with non-primitive types, but cosmos sdk queries are all present in the URL or GET query params and using primitives. Using primitives is also simpler for integrations since they don't need to know about the Coin type, and can pass in a value in the form that they expect from other queries.

Reply:
GETs don't have request bodies but only POSTs.